process.rb

```diff
@@ -280,6 +299,12 @@ def slugify(word)
       totals[:most_expensive_races] = totals[:race_totals].sort_by {|v| -v[:amount]}[0..2]
       totals.delete(:race_totals)
 
+      # Get the top 3 small contribution proprtions
+      unless totals[:small_proportion].nil?
+        totals[:largest_small_proprtion] = totals[:small_proportion].sort_by {|v| -v[:proportion]}[0..2]
+      end
```

Comment:
I wonder if we will also want to filter out candidates without a certain threshold of contributions. Like, if someone has only raised a couple hundred bucks, do we show them atop the leaderboard? I wouldn't object to it. I guess we'll see what the data looks like.

Reply:
Good thought.  Not clear what the right measure should be.  Maybe there should be a certain threshold number of itemized.